### ISSUE TITLE
ARM-based REST API support in developer portal migrate scripts

### ISSUE THREAD

**Issue by lordldx** [Azure/api-management-developer-portal#953]
The current [migrate script](https://github.com/Azure/api-management-developer-portal/blob/master/scripts.v2/migrate.js) requires access to the management api. 
Most of our customers refuse this because it gives us admin level access without any auditing.
Can the migrate script be made to support the [ARM-based REST API](https://docs.microsoft.com/en-us/rest/api/apimanagement/?WT.mc_id=Portal-Microsoft_Azure_ApiManagement)?

**Comment by azaslonov** [comment]
Hi @lordldx, I think it should work even now if you put "bearer" token instead of SAS token and provide a full ARM-based management API URL. The only problem, for now, is that we don't have out of the box script (it would invoke AAD login dialog) to get "bearer" token, probably this is something we'll do in the future. Meanwhile, we greatly appreciate contributions :)

**Comment by mikebudzynski** [comment]
@lordldx We'll soon publish documentation for respective ARM API endpoints at https://docs.microsoft.com/rest/api/apimanagement/

**Comment by lordldx** [comment]
That's great news @mikebudzynski , thanks!
Looking forward to it :-)

**Comment by mikebudzynski** [comment]
It's already live: https://docs.microsoft.com/en-us/rest/api/apimanagement/2019-12-01/contenttype/createorupdate

We'll see if the script can be refactored to use ARM APIs, but at this moment it's a lower priority work item.

**Comment by lordldx** [comment]
Thanks @mikebudzynski 
Docs look great.

I think I'll wait on the updated script though. It's a bit hard to understand all if you don't know how the dev portal is implemented.

**Comment by azaslonov** [comment]
ARM-based scripts added. Please see [this folder](https://github.com/Azure/api-management-developer-portal/tree/master/scripts.v3).

**Comment by SteppingRazor** [comment]
Hi @azaslonov, thanks for the scripts. For my own purposes I'm rewriting them to C#. I have a question about the publish though. You wrote in the comment that new version of API is not in production yet. Where can I find source for this and where can I follow the progress for the new api version? Is this new version coming in 2021?

**Comment by azaslonov** [comment]
Hi @SteppingRazor, there is no public sources for this, I hope to see the new version out sooner, but with all the upcoming holidays it may indeed slip to 2021. Please watch for APIM announcements.
